server/src/rooms/utils/CommsSetup.ts

```diff
@@ -92,28 +92,28 @@ function setUpVoiceListener(room: Room<MyRoomState>) {
   });
 }
 
-function setUpRoomUserListener(room: Room<MyRoomState>) {
-  room.onMessage("player_joined", (client, message) => {
-    //get all currentplayer's session ids
-    // not used as room userlistener anymore
-    // room.broadcast("new_player", [allPlayers]);
-    const allPlayers = room.clients.map((client) => {
-      return room.state.players.get(client.sessionId).userName;
-    });
-    allPlayers.filter((player) => player !== undefined);
-    room.broadcast("new_player", [allPlayers]);
-  });
+// function setUpRoomUserListener(room: Room<MyRoomState>) {
```

Comment:
I dont think can remove this for now, GameUi need these to update the username list which is not working. Will change the event name to follow the standard.

Reply:
got it ill bring it back thanks for the comment